From b881e0b2ba9cf1a4aa351a1c1ea90b1e1776ce21 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Mon, 12 Aug 2019 15:37:36 +0200
Subject: [PATCH] numpy/random/setup.py: remove the detection of x86 targets
 via uname()

This was badly breaking compilation for ARM targets (and possibly
other non-x86 targets); if -msse2 is desirable for x86 builds it has
to be passed through another channel.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 numpy/random/setup.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/numpy/random/setup.py b/numpy/random/setup.py
index a1bf3b8..60fb534 100644
--- a/numpy/random/setup.py
+++ b/numpy/random/setup.py
@@ -49,11 +49,6 @@ def configuration(parent_package='', top_path=None):
     elif not is_msvc:
         # Some bit generators require c99
         EXTRA_COMPILE_ARGS += ['-std=c99']
-        INTEL_LIKE = any(arch in platform.machine() 
-                         for arch in ('x86', 'i686', 'i386', 'amd64'))
-        if INTEL_LIKE:
-            # Assumes GCC or GCC-like compiler
-            EXTRA_COMPILE_ARGS += ['-msse2']
 
     # Use legacy integer variable sizes
     LEGACY_DEFS = [('NP_RANDOM_LEGACY', '1')]
